packages/orbit-themer/src/Tabs/ColorTab.jsx

```diff
@@ -0,0 +1,86 @@
+import styled from "styled-components";
+import React from "react";
+import equals from "ramda/src/equals";
+import path from "ramda/src/path";
+import ChevronDown from "@kiwicom/orbit-components/lib/icons/ChevronDown";
+import ChevronRight from "@kiwicom/orbit-components/lib/icons/ChevronRight";
+import InformationCircle from "@kiwicom/orbit-components/lib/icons/InformationCircle";
+import Badge from "@kiwicom/orbit-components/lib/Badge";
+import Heading from "@kiwicom/orbit-components/lib/Heading";
+
+import { DEFAULT_COLORS } from "../consts";
+import ColorContext from "../ColorContext";
+
+const StyledColorTab = styled.div`
+  font-size: 14px;
+`;
+
+const StyledColorTabHeader = styled.div`
+  cursor: pointer;
+  padding: 14px 12px;
+  border-radius: 6px;
+  position: relative;
+  background: ${({ opened, theme }) => opened && theme.orbit.paletteCloudDark};
+
+  &:hover {
+    background: ${({ opened, theme }) =>
+      opened ? theme.orbit.paletteCloudDark : theme.orbit.paletteCloudLight};
+  }
+`;
+
+const StyledColorTabIcon = styled.div`
+  position: absolute;
+  top: 50%;
+  margin-top: -8px;
+  right: 12px;
+  color: #bac7d5;
```

Comment:
I think we have a token for that value, check the palette inside design tokens ;)